gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -606,6 +652,12 @@ def _wait_for_network_function_operation_completion(self, context,
             elif operation.lower() == 'update':
                 raise NodeInstanceUpdateFailed()
 
+        if (self._get_service_type(
+                context.current_profile) == pconst.LOADBALANCER and
+                    operation.lower()=='create'):
+            self._create_policy_target_for_vip(context)
+
+
```

Comment:
How about delete? I am not sure how this works with lbaasv2. It uses the same vip for multiple listeners.